packages/react-reconciler/src/ReactFiberNewContext.js

```diff
@@ -736,6 +800,8 @@ function readContextForConsumer<T>(
       context: ((context: any): ReactContext<mixed>),
       memoizedValue: value,
       next: null,
+      compare: ((compare: any): ContextCompare<mixed, mixed> | null),
```

Comment:
kind of an unsafe cast. You're assigning undefined here. You used the `!= null` form elsewhere when checking compares existence so this is probably working.

I'd maybe just fork this entire function though because we generally don't want to allocate the object with extra unused properties

Reply:
Updated to fork and resolved everywhere to `null` instead of `void`. We still need to check `!= null` before using `compare` since we need the standard ContextDependencies to work correctly even with the flag enabled (preserve regular `useContext`). Otherwise we'd need to fork additional functions in the propagation path.